src/test/ringdelaybuffer_test.cpp

```diff
@@ -49,35 +48,38 @@ TEST_F(RingDelayBufferTest, ReadWriteNoDelayTest) {
 
     mixxx::SampleBuffer output(numSamples);
 
+    std::span<CSAMPLE> outputSpan = output.span();
+    std::span<CSAMPLE> inputBufferSpan = mixxx::spanutil::spanFromPtrLen(inputBuffer, numSamples);
```

Comment:
Build fails because `spanFromPtrLen` can only create a `std::span<const CSAMPLE>` since the buffer it observes is also `const`. Either of the below suggestions would work. I prefer the `auto` version since the type is clear and the line is already somewhat long. 

```suggestion
    std::span<const CSAMPLE> inputBufferSpan = mixxx::spanutil::spanFromPtrLen(inputBuffer, numSamples);
```

```suggestion
    auto inputBufferSpan = mixxx::spanutil::spanFromPtrLen(inputBuffer, numSamples);
```

Reply:
Auto looks better in this situation to me too. Thank you.